venv.go

```diff
@@ -136,10 +137,11 @@ func (c VenvCommand) Run() (string, string, error) {
 		}
 
 		if err := cmd.Wait(); err != nil {
-			return "", "", errors.Wrap(err, "unable to complete command")
+			exitError, _ := err.(*exec.ExitError)
+			return "", "", errors.Wrap(err, "unable to complete command"), exitError.ExitCode()
 		}
 
-		return "", "", nil
+		return "", "", nil, 0
```

Comment:
At this point we should consider refactoring instead of bolting on another parameter.
This is basically returning all the same fields as `AnsibleRunOutput`. you could move that here, rename it to `VenvRunResult` and just thread that through the code instead of building, packing, unpacking, then packing up again.